ets-demo/etsdemo/tests/test_main.py

```diff
@@ -59,7 +59,7 @@ def test_main(self):
             main_module.main()
             console_output = mocked_io.getvalue()
 
-        self.assertEqual(console_output, "")
+        self.assertNotIn("Found 0 resource(s).", console_output)
```

Comment:
Where exactly does "Found 0 resource(s)." come from?  The idea is that no loggings should be emitted, does this mean that if any log is emitted, this will always be in the console_output?

Reply:
Offline discussion:
The message comes from https://github.com/enthought/traitsui/blob/9ea583c7d69a32dc4a6d7896529274161c0cab70/ets-demo/etsdemo/main.py#L109

The description of this PR is a bit misleading: Not only that we don't want the test to fail because of warnings from code outside of our control, this particular test is targeting the logging settings, so we don't want it to fail even if there are warnings from our code. Even though such warnings are undesirable, they are not the objective of this test.
I fixed the description.

Thanks @aaronayres35!